pkg/oci/utils.go

```diff
@@ -37,6 +37,34 @@ var (
 
 	// BundlePathKey is the annotation key to fetch the OCI configuration file path.
 	BundlePathKey = "com.github.containers.virtcontainers.pkg.oci.bundle_path"
+
+	// ContainerTypeKey is the annotation key to fetch container type.
+	ContainerTypeKey = "com.github.containers.virtcontainers.pkg.oci.container_type"
+
+	// CRIOContainerTypeKey is the annotation key to fetch CRIO container type.
+	CRIOContainerTypeKey = "crio/container_type"
```

Comment:
This is defined by CRI-O, not us.... Take a look here: https://github.com/kubernetes-incubator/cri-o/blob/master/server/sandbox_run.go#L263, an here: https://github.com/kubernetes-incubator/cri-o/blob/master/server/container_create.go#L541